avutil/avutil.go

```diff
@@ -980,6 +1009,110 @@ func (f *Frame) PacketDuration() int64 {
 	return int64(C.av_frame_get_pkt_duration(f.CAVFrame))
 }
 
+func (f *Frame) SideData(dataType FrameSideDataType) *FrameSideData {
+	cSideData := C.av_frame_get_side_data(f.CAVFrame, (C.enum_AVFrameSideDataType)(dataType))
+	if cSideData == nil {
+		return nil
+	}
+
+	return NewFrameSideDataFromC(unsafe.Pointer(cSideData))
+}
+
+type FrameSideData struct {
+	CFrameSideData *C.AVFrameSideData
+}
+
+func NewFrameSideDataFromC(cFrameSideData unsafe.Pointer) *FrameSideData {
+	return &FrameSideData{
+		CFrameSideData: (*C.AVFrameSideData)(cFrameSideData),
+	}
+}
+
+func (sd *FrameSideData) Size() int {
+	return int(sd.CFrameSideData.size)
+}
+
+func (sd *FrameSideData) Type() FrameSideDataType {
+	return FrameSideDataType(sd.CFrameSideData._type)
+}
+
+func (sd *FrameSideData) MotionVector(n int) *MotionVector {
```

Comment:
I see you didn't follow my advice for `MotionVectors() []*MotionVector`.
Also, you're not using `go_motion_vector_size` anymore.
How will you discover the number of motion vectors in go?